app/Entirety/requirements.txt

```diff
@@ -10,7 +10,7 @@ django-compressor~=4.1
 django-libsass==0.9
 filip~=0.2.5
 mozilla_django_oidc==2.0.0
-pydantic==1.10.2
+pydantic==1.*
```

Comment:
This might also install version 1.10.2. Since we know that 1.10.13 is working, how about changing it to `pydantic>=1.10.13`?

Reply:
How about this? `pydantic>=1.10.13,<2.*`. `<2.*` is only ensure the current Entirety can work. After that we can still do the pydantic v2 migration